config/language/blacklist.js

```diff
@@ -0,0 +1,20 @@
+
+/**
+  This blacklist lists all the language codes we exclude for import in to placeholder.
+
+  The whosonfirst dataset contains many disused and rarely-used languages which
+  can cause issues when the source data has been machine transiliterated.
+
+  The list is non-exhaustive and was originally sourced from wikipedia and various
+  online sources, I aimed to include the most commonly spoken languages worldwide.
```

Comment:
this line probably needs an update for the blacklist :)

Reply:
fixed!